code/modules/client/client procs.dm

```diff
@@ -392,6 +395,14 @@
 		if(ckey != query_cid.item[1])
 			related_accounts_cid.Add("[query_cid.item[1]]")
 
+	var/admin_rank = "Player"
+	if(holder)
+		admin_rank = holder.rank
+	// Admins don't get slammed by this, I guess
+	else
+		if(check_randomizer())
```

Comment:
This proc also returns true if it's just making the client reconnect for a cid change check, returning here prevents logging both this connection and the reconnect for no reason.